batch/batch/worker/worker.py

```diff
@@ -2104,15 +2104,6 @@ def __str__(self):
         )
 
 
-@contextmanager
```

Comment:
Was this not used at all?

Reply:
Not that I could find by grepping around.